students/ErvinBesic/module-2-html/exercises/03-recipe/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="author" content="Ervin Besic">
+    <meta name="description" content="Vjezba 3-Apple pie recipe">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Apple pie recipe</title>
+</head>
+
+<body>
+    <img src="apple-pie.jpg" alt= "Apple pie" width="300" height="200">
+    <h1>Apple pie</h1>
+    <h2>Total Time</h2>
+    <p>Prep: 20 min. Bake: 45 min.</p>
+    <h2>Makes</h2>
+    <p>8 servings</p>
+    <h2>Ingredients</h2>
+    <ul>
+        <li>1/2 cup sugar</li>
+        <li>1/2 cup packed brown sugar</li>
+        <li>3 tablespoons all-purpose flour</li>
+        <li>1 teaspoon ground cinnamon</li>
+        <li>1/4 teaspoon ground ginger</li>
+        <li>1/4 teaspoon ground nutmeg</li>
+        <li>6 to 7 cups thinly sliced peeled tart apples</li>
+        <li>1 tablespoon lemon juice</li>
+        <li>Pastry for double-crust pie (9 inches)</li>
+        <li>1 tablespoon butter</li>
+        <li>1 large egg white</li>
+        <li>Additional sugar</li>
+    </ul>
+    <h2>Directions</h2>
+    <p>
+        In a small bowl, combine the sugars, flour and spices; set aside. In a large bowl, toss apples with lemon juice.
+        Add sugar mixture; toss to coat.<br>
+        Line a 9-in. pie plate with bottom crust; trim even with edge. Fill with apple mixture; dot with butter. Roll
+        remaining crust to fit top of pie; place over filling. Trim, seal and flute edges. Cut slits in crust.<br>
+        Beat egg white until foamy; brush over crust. Sprinkle with sugar. Cover edges loosely with foil.<br>
+        Bake at 375° for 25 minutes. Remove foil and bake until crust is golden brown and filling is bubbly, 20-25
+        minutes longer. Cool on a wire rack.<br>
+    </p>
+    <h2>Nutrition Facts</h2>
+    <p>1 piece: 414 calories, 16g fat (7g saturated fat), 14mg cholesterol, 227mg sodium, 67g carbohydrate (38g sugars,
+        2g fiber), 3g protein.</p>
+    <p> &#169 Taste of Home</p>
```

Comment:
HTML entiteti završavaju sa `;`. Copyright simbol također možeš pisati i kao ˙&copy;`, osobno mi je taj lakše zapamtiti.

```suggestion
    <p> &#169; Taste of Home</p>
```